client/src/components/tabs.js

```diff
@@ -53,15 +53,15 @@ export default class Tabs extends React.Component{
                     { ! country_name ?
                     <>
                         <p className="is-size-7">
-                            Please use the dropdown in the header for country specific comparisons and projections. 
+                            Please use the dropdown in the header for country-specific comparisons and projections. 
                             
                         </p>
                         
                     </>
                     :
                     <></>
                     }
-                    <p className="is-size-7">We support countries with over 1 million population and 10 confirmed cases.</p>
```

Comment:
I'm concerned that "support" can be misread as "care about", which is obviously not what is intended, but I figured it was easy to change to something pretty-much equivalent.

Reply:
Very valid concern Dave